### PR TITLE
v4 -> v6. v6 -> v7

### PR DESCRIPTION
**Pull request by pierreneter**

### CODE REVIEW

File: locale/en/download/package-manager.md [hunk 1]
```diff
@@ -114,16 +114,16 @@ Node.js is available from the [NodeSource](https://nodesource.com) Enterprise Li
 
 Note that the Node.js packages for EL 5 (RHEL5 and CentOS 5) depend on the **[EPEL](https://fedoraproject.org/wiki/EPEL)** repository being available. The setup script will check and provide instructions if it is not installed.
 
-Run as root on RHEL, CentOS or Fedora, for Node.js v4 LTS Argon:
+Run as root on RHEL, CentOS or Fedora, for Node.js v6 LTS:
```
Comment (rvagg) [inline locale/en/download/package-manager.md:117]: `for Node.js v6 LTS "Boron"`

Comment (rvagg) [reply locale/en/download/package-manager.md:117]: actually, scrap that, it's not used elsewhere in the doc so this change is fine for now

**Comment by rvagg** [comment]
lgtm

**Comment by lpinca** [comment]
Can you please use a more descriptive commit message? Apart from that LGTM.

**Comment by fhemberger** [comment]
Updated the commit message, thank you!

**Pull request merged**